nautobot/utilities/forms/forms.py

```diff
@@ -57,6 +63,27 @@ class BootstrapMixin(forms.BaseForm):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
+        content_type = None
+        form_class = None
+        try:
+            content_type = get_content_type_string(self.model)
+            form_class = get_form_for_model(self.model, "Filter")
+        except AttributeError:
+            pass
+
+        if registry["plugin_filter_extensions"].get(content_type) and self.__class__ is form_class:
+            for filter_form in registry["plugin_filter_extensions"][content_type]:
+                if not filter_form().filter_form():
+                    continue
+                for filter_name, form in filter_form().filter_form().items():
```

Comment:
Would be preferable as I mentioned in my other comment to store `filter_form()` return value directly in the registry so it doesn't need to be called here, but at the very least let's use a local variable so it doesn't need to be called twice.